lib/streamlit/elements/multiselect.py

```diff
@@ -214,6 +215,15 @@ def _check_and_convert_to_indices(
                     default_values = [default_values]
                 else:
                     default_values = list(default_values)
+            if len(default_values) != 0 and isinstance(default_values[0], Enum):
+                str_default_values = list(map(lambda enum: str(enum), default_values))
+                mapped_opt_keys = list(map(lambda enum: str(enum), opt))
```

Comment:
These would probably be better as list comprehensions for consistency/idiomaticity